data/campaigns/Descent_Into_Darkness/scenarios/07a_A_Small_Favor.cfg

```diff
@@ -770,7 +770,6 @@
         [/kill]
         [endlevel]
             result=victory
-            linger_mode=no
```

Comment:
Leave the DiD ones as they were originally please, including the ones for Endless Night.